Pocket/Pocket/Models/Query.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Query.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 6/5/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public class Query: PocketRequestProtocol {
+    public var network = ""
+    public var query = Dictionary<AnyHashable, Any>()
```

Comment:
Don't use this syntax. 

Use: `[AnyHashable: Any]`. 

Make this change everywhere.